src/lib/AnnouncementHeader.svelte

```diff
@@ -26,29 +26,27 @@
 
   .container {
     display: flex;
+    flex-direction: column;
```

Comment:
I don't think we need a flexbox for this. `text-align: center` should do the trick for use while being a lot simpler. It'd need two divs though. One for the icon, one for the text.